datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxClusterRequest.java

```diff
@@ -108,4 +110,12 @@ public boolean isEnableRangerRaz() {
     public void setEnableRangerRaz(boolean enableRangerRaz) {
         this.enableRangerRaz = enableRangerRaz;
     }
+
+    public boolean isEnableMediumDutyHA() {
```

Comment:
I don't understand why we need this? Shouldn't you just check if the cluster shape is Medium Duty?

Reply:
yeah, good point.